_data/current.yml

```diff
@@ -281,7 +281,14 @@
   url: http://agiletestingdays.com
   twitter: AgileTD
   status: CFP Open until April 2018
-
+  
```

Comment:
This is in the wrong place. Need to move it down in the list so everything is in Chronological order.

Reply:
Hi, I put it after AgileTD and before SQA Days... Which seems to be in chronological order... Am I missing something else? :) Thanks